src/shared/components/common/password-input.tsx

```diff
@@ -98,7 +100,7 @@ class PasswordInput extends Component<PasswordInputProps, PasswordInputState> {
                 autoComplete={isNew ? "new-password" : "current-password"}
                 onInput={onInput}
                 value={value}
-                required
+                required={required !== false}
```

Comment:
Are you using `required !== false` because you want it to default to true even if it's undefined?